nifi-registry-core/nifi-registry-provider-api/src/main/java/org/apache/nifi/registry/hook/EventFieldName.java

```diff
@@ -26,6 +26,6 @@
     EXTENSION_BUNDLE_ID,
     VERSION,
     USER,
-    COMMENT;
-
+    USER_GROUP,
+    COMMENT
```

Comment:
I'n not sure we should reuse the EventFieldName.USER for the the user that was created. In other contexts it indicates the user that performed the operation (speaking of which, maybe we want that information for tenant CRUD events?)

I would suggest adding new EventFieldNames that align with the the tenant data model, such as `TENANT_ID` and `TENANT_IDENTITY` that can be used for all tenant types, or if we want to break it down into users and groups, `USER_ID`, `USER_IDENTITY`, `USER_GROUP_ID`, `USER_GROUP_IDENTITY`. What do you think?

Reply:
I think USER_ID, USER_IDENTITY, USER_GROUP_ID, USER_GROUP_IDENTITY more closely matches the terminology in object, methods of the tenant resource but don't have very strong feelings either way.